### ISSUE TITLE
Disable eureka ship disable via GUI as if helms are broken

### ISSUE THREAD

**Issue by awesomebossdj7** [ValkyrienSkies/Eureka#61]
make a option in the ship helm to be able to ragdoll, so you could put a ship onto another ship 

ex: a spaceship lands on a space station and ragdolls so the space station can move

**Comment by awesomebossdj7** [comment]
landing would be a better word lol

**Comment by ThePlasticPotato** [comment]
Out of scope of Eureka, and I'm not entirely sure what that means anyways-

**Comment by awesomebossdj7** [comment]
> Out of scope of Eureka, and I'm not entirely sure what that means anyways-

what i mean is that basically turn off flight button

**Comment by Rubydesic** [comment]
You mean a button to disable the stabilization provided by the helm (i.e. what happens when you break the helm?)

**Comment by awesomebossdj7** [comment]
> You mean a button to disable the stabilization provided by the helm (i.e. what happens when you break the helm?)

yes exactly that, useful for like turning your ship from a airship to a boat and other things that require turning off flight, like landing.
